### PR TITLE
Add support for TEUR

### PR DESCRIPTION
**Pull request by Alteway**
TRC20 token "TEUR" added

### CODE REVIEW

File: packages/cryptoassets/data/trc20.js [hunk 1]
```diff
@@ -4,5 +4,6 @@ module.exports = [
   ["TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9","JST","JUST GOV","TCFLL5dx5ZJdKnWuesXxi1VPwjLVmWZZy9",18],
   ["TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT","USDJ","JUST Stablecoin","TMwFHYXLJaRUPeW6421aqXL4ZEzPRFGkGT",18],
   ["TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS","KLV","Klever","TVj7RNVHy6thbM7BWdSe9G6gXwKhjhdNZS",6],
-  ["TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh","SEED","SEED","TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh",6]
+  ["TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh","SEED","SEED","TBwoSTyywvLrgjSgaatxrBhxt3DGpVuENh",6],
+  ["TZJmk51TP3YEETw7phRP581A7H3VzLDFsa","TEUR,"EUR AlteWay Token","TZJmk51TP3YEETw7phRP581A7H3VzLDFsa",4]
```
Comment (haammar-ledger) [inline packages/cryptoassets/data/trc20.js:8]: Missing `"`

**Review by ghost — CHANGES_REQUESTED** [review]

File: packages/cryptoassets/data/trc20.js (hunk 1 above)
Comment (Alteway) [reply packages/cryptoassets/data/trc20.js]: Added "

**Review by Alteway — COMMENTED** [review]
Added "

**Review by Alteway — COMMENTED** [review]
Fixed

**Review by gre — APPROVED** [review]

**Pull request merged**